core/context.h

```diff
@@ -71,6 +71,9 @@ OIDN_NAMESPACE_BEGIN
         if (getEnvVarOrDefault("OIDN_DEVICE_METAL", 1))
           OIDN_INIT_STATIC_MODULE(device_metal);
       #endif
+      #if defined(OIDN_METAL_IOS)
+        getEnvVarOrDefault("OIDN_METAL_IOS", 1);
+      #endif
```

Comment:
We shouldn't add a separate environment variable for this. It's still the same Metal device, just the OS is different.